guides/common/assembly_installing-satellite-server-disconnected.adoc

```diff
@@ -22,6 +22,20 @@ include::modules/proc_downloading-the-binary-dvd-images.adoc[leveloffset=+1]
 
 include::modules/proc_configuring-the-base-operating-system-with-offline-repositories.adoc[leveloffset=+1]
 
+include::modules/con_installing-and-configuring-the-team-insights-on-premises-advisor-engine-in-a-disconnected-environment.adoc[leveloffset=+1]
```

Comment:
We need to drop the term `insights on premises' even from the file names for alignment.